lib/plugin/stepByStepReport.js

```diff
@@ -20,6 +20,7 @@ const defaultConfig = {
   fullPageScreenshots: false,
   output: global.output_dir,
   screenshotsForAllureReport: false,
+  disableScreenshotOnFail: false,
```

Comment:
I would say this must set to true as there probably not a real need to have screenshots after failure

Reply:
I wanted to keep the functionality as it is, but i think also it would make sense to change it to true as default.